website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -87,6 +87,55 @@ In Consul v1.15 and higher:
 
    </CodeBlockConfig>
 
+#### `service-defaults` Overrides for Upstream Peered Services
+
+In Consul 1.14.x, `service-defaults` upstream [`overrides`](/consul/docs/connect/config-entries/service-defaults#overrides) would apply to a both local and peered services as long as the `name` field matched.
+Consul 1.15.0 introduces a compatibility mode to maintain this behavior as long as [`peer`](/consul/docs/connect/config-entries/service-defaults#peer) is not set in any override.
```

Comment:
```suggestion
Consul 1.15.0 is backward compatible with 1.14 if the [`peer`](/consul/docs/connect/config-entries/service-defaults#peer) is not set in any override.
```
Just to make this simpler and remove the suggestion that we're implementing a new mode that the user has to keep track of. I understand this just to be more of a behavior